interview-questions.md

```diff
@@ -8,42 +8,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer: Object-oriented programming focuses on using key/value pairs whereas functional programming puts an emphasis on using higher order functions?
+
+Researched answer: OOP programming is more abstract because it has a focus on operating on keys directly instead of functional programming's declarative approach that requires explicit functions. As such, they have different methods for storing and handling data.
```

Comment:
nice. additional content could be sharing some websites that use those programming styles